features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/Create/CreateCustomRulePolicy.js

```diff
@@ -16,6 +16,7 @@
  * under the License.
  */
 import React, { Component } from 'react';
+import {withRouter} from 'react-router-dom'
```

Comment:
IMHO we don't need `withRouter` here since we are rendering this component through a Route?

```javascript
<Route path={"/policies/custom_rules/create"} render={props => (<CreateCustomRulePolicy/>)}/> 
```

Reply:
this is used to route between  pages using history push